persist/fs/seek.go

```diff
@@ -64,9 +64,9 @@ type seeker struct {
 	summariesFdWithDigest      digest.FdWithDigestReader
 	digestFdWithDigestContents digest.FdWithDigestContentsReader
 
-	dataFd     *os.File
 	dataReader *bufio.Reader
```

Comment:
bufio.Reader no longer necessary with the mmap yeah?  Can just make this a `*bytes.Reader`.